Blockzilla/UIConstants.swift

```diff
@@ -41,6 +41,7 @@ struct UIConstants {
         static let navigationTitle = UIConstants.Photon.Grey10
         static let overlayBackground = UIColor(white: 0, alpha: 0.8)
         static let progressBar = UIColor(rgb: 0xC86DD7)
+		static let searchSuggestion = UIConstants.Photon.Grey10.withAlphaComponent(0.6)
```

Comment:
It so weird. Something goes wrong with indentation when the code is pushed to git. It looks ok in XCode
<img width="884" alt="screen shot 2019-01-01 at 2 07 27 pm" src="https://user-images.githubusercontent.com/26641473/50575561-aee24800-0dce-11e9-9efe-ce1da0eee9ed.png">

Reply:
I'm trying to figure out what causes it.

Reply:
Yeah this one in specific I wouldn't worry about too much since it also shows normally in my Xcode. The rest of the nits are showing up in my Xcode, so if you fix those that should be sufficient!